media-libs/opencv/opencv-4.9.0.ebuild

```diff
@@ -0,0 +1,1017 @@
+			/usr/include/opencv4/opencv2/cudev/util/detail/type_traits.hpp
+			/usr/include/opencv4/opencv2/cudev/util/limits.hpp
+			/usr/include/opencv4/opencv2/cudev/util/saturate_cast.hpp
+			/usr/include/opencv4/opencv2/cudev/util/simd_functions.hpp
+			/usr/include/opencv4/opencv2/cudev/util/tuple.hpp
+			/usr/include/opencv4/opencv2/cudev/util/type_traits.hpp
+			/usr/include/opencv4/opencv2/cudev/util/vec_math.hpp
+			/usr/include/opencv4/opencv2/cudev/util/vec_traits.hpp
+			/usr/include/opencv4/opencv2/cudev/warp/detail/reduce.hpp
+			/usr/include/opencv4/opencv2/cudev/warp/detail/reduce_key_val.hpp
+			/usr/include/opencv4/opencv2/cudev/warp/reduce.hpp
+			/usr/include/opencv4/opencv2/cudev/warp/scan.hpp
+			/usr/include/opencv4/opencv2/cudev/warp/shuffle.hpp
+			/usr/include/opencv4/opencv2/cudev/warp/warp.hpp
+			# [contribcvv]
+			/usr/include/opencv4/opencv2/cvv.hpp
+			/usr/include/opencv4/opencv2/cvv/call_meta_data.hpp
+			/usr/include/opencv4/opencv2/cvv/cvv.hpp
+			/usr/include/opencv4/opencv2/cvv/debug_mode.hpp
+			/usr/include/opencv4/opencv2/cvv/dmatch.hpp
+			/usr/include/opencv4/opencv2/cvv/filter.hpp
+			/usr/include/opencv4/opencv2/cvv/final_show.hpp
+			/usr/include/opencv4/opencv2/cvv/show_image.hpp
+			# [contribdnn]
+			/usr/include/opencv4/opencv2/dnn.hpp
+			/usr/include/opencv4/opencv2/dnn/all_layers.hpp
+			/usr/include/opencv4/opencv2/dnn/dict.hpp
+			/usr/include/opencv4/opencv2/dnn/dnn.hpp
+			/usr/include/opencv4/opencv2/dnn/dnn.inl.hpp
+			/usr/include/opencv4/opencv2/dnn/layer.details.hpp
+			/usr/include/opencv4/opencv2/dnn/layer.hpp
+			/usr/include/opencv4/opencv2/dnn/shape_utils.hpp
+			/usr/include/opencv4/opencv2/dnn/utils/debug_utils.hpp
+			/usr/include/opencv4/opencv2/dnn/utils/inference_engine.hpp
+			/usr/include/opencv4/opencv2/dnn/version.hpp
+			/usr/include/opencv4/opencv2/dnn_superres.hpp
+			# [contribhdf]
+			/usr/include/opencv4/opencv2/hdf.hpp
+			/usr/include/opencv4/opencv2/hdf/hdf5.hpp
+
+			/usr/include/opencv4/opencv2/mcc.hpp
+			/usr/include/opencv4/opencv2/mcc/ccm.hpp
+			/usr/include/opencv4/opencv2/mcc/checker_detector.hpp
+			/usr/include/opencv4/opencv2/mcc/checker_model.hpp
+
+			/usr/include/opencv4/opencv2/text.hpp
+			/usr/include/opencv4/opencv2/text/erfilter.hpp
+			/usr/include/opencv4/opencv2/text/ocr.hpp
+			/usr/include/opencv4/opencv2/text/swt_text_detection.hpp
+			/usr/include/opencv4/opencv2/text/textDetector.hpp
+
+			# [qt5,qt6]
+			/usr/include/opencv4/opencv2/viz.hpp
+			/usr/include/opencv4/opencv2/viz/types.hpp
+			/usr/include/opencv4/opencv2/viz/viz3d.hpp
+			/usr/include/opencv4/opencv2/viz/vizcore.hpp
+			/usr/include/opencv4/opencv2/viz/widget_accessor.hpp
+			/usr/include/opencv4/opencv2/viz/widgets.hpp
+
+			/usr/include/opencv4/opencv2/wechat_qrcode.hpp
+		) # }}}
+	fi
+	if multilib_is_native_abi && use python; then
+		python_foreach_impl cmake_src_install
+		python_foreach_impl python_optimize
+	else
+		cmake_src_install
+	fi
+}
+
+multilib_src_test() {
```

Comment:
Please don't do newline after function starts.

Also, multilib_src_test should go before multilib_src_install to reflect phase execution order.